docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -5,9 +5,17 @@
 <titleabbrev>Synthetic monitoring</titleabbrev>
 ++++
 
-The {uptime-app} in {kib} lets you monitor user journeys and full page loads using our browser monitor,
-and also monitor the availability and response times of applications and services using our lightweight HTTP/S,
-TCP, and ICMP monitors.
+Synthetic monitoring extends traditional end-to-end testing techniques because it allows your tests to run continuously on the cloud.
+With Elastic's Synthetic Monitoring product, you can reuse the same journeys that you've used to validate the software on your machine
+to assert that your application continues to work after a deployment.
```

Comment:
Rethinking about this sentence, it feels unbalanced to me. Most of it's weight is in the middle and by the time you get to the end you forgot the beginning (which makes the point that you can reuse journeys).

I think I'd actually change to:

> With Elastic's Synthetic Monitoring product, you can assert that your application continues to work after a deployment by reusing the same journeys that you've used to validate the software on your machine.

I know I was the person who suggested the current version, but just thought about it once again and it felt like it could be improved. This is just a nit though, feel free to ignore this comment if you prefer the current version.

Sorry about suggesting a change to this paragraph once again.

Reply:
I agree. Leading with the _why_ can make the reader confident that it's worth reading to the end to uncover the _how_. 

>Sorry about suggesting a change to this paragraph once again.

I do this all the time. 🙈  Iteration is key!